invenio_circulation/ext.py

```diff
@@ -38,15 +38,16 @@ def __init__(self, app=None):
     def init_app(self, app):
         """Flask application initialization."""
         self.init_config(app)
-        app.config.setdefault("RECORDS_REST_ENDPOINTS", {})
 
-        app.config["CIRCULATION_REST_ENDPOINTS"].setdefault("loanid", {})[
-            "links_factory_imp"
-        ] = app.config["CIRCULATION_LOAN_LINKS_FACTORY"]
+        # init CIRCULATION_REST_ENDPOINTS
+        k = CIRCULATION_LOAN_PID_TYPE
```

Comment:
`k`: a single character variable is not accepted in `reroils`, I do now know about your guidelines.